qa/acceptance-tests/src/test/java/io/camunda/it/auth/ProcessAuthorizationIT.java

```diff
@@ -176,13 +176,30 @@ private static DeploymentEvent deployResource(
 
   private static void waitForProcessesToBeDeployed(
       final CamundaClient camundaClient, final int expectedCount) {
-    Awaitility.await("should deploy processes and import in Operate")
+    Awaitility.await("should deploy processes and make them searchable")
         .atMost(Duration.ofSeconds(15))
         .ignoreExceptions() // Ignore exceptions and continue retrying
         .untilAsserted(
             () -> {
               final var result = camundaClient.newProcessDefinitionSearchRequest().send().join();
               assertThat(result.items().size()).isEqualTo(expectedCount);
             });
+
+    // also wait for the start form to be searchable, as tests depend on that
+    final var startFormProcessDefinitionKey =
+        getProcessDefinitionKey(camundaClient, PROCESS_DEFINITION_ID_WITH_START_FORM);
+
+    Awaitility.await("should deploy start form and make it searchable")
+        .atMost(Duration.ofSeconds(15))
+        .ignoreExceptions() // Ignore exceptions and continue retrying
+        .untilAsserted(
+            () -> {
+              final var result =
+                  camundaClient
+                      .newProcessDefinitionGetFormRequest(startFormProcessDefinitionKey)
+                      .send()
+                      .join();
+              assertThat(result).isNotNull();
```

Comment:
🔧 I would recommend using `assertThat(CompletableFuture).succeedsWithin` or something like this. This would make sure that if this times out because the future is always failing (so it throws an exception on join), we will see the error message in the failure, as opposed to just ignoring it right now.

Reply:
That makes sense to me, will adjust it.